lib/runners/processor/actionlint.rb

```diff
@@ -0,0 +1,83 @@
+module Runners
+  class Processor::Actionlint < Processor
+    SCHEMA = _ = StrongJSON.new do
+      extend Schema::ConfigTypes
+
+      # @type self: SchemaClass
+      let :config, base(
+        target: target,
+
+        # Native options
+        ignore: enum?(string, array(string)),
```

Comment:
```suggestion
        ignore: one_or_more_strings?,
```

This useful method is defined at https://github.com/sider/runners/blob/ee5e32a7b86a5911293b112d7d0aab8c5cf66c8c/lib/runners/schema/config.rb#L6-L8

Would you be willing to add this in-house rule to [goodcheck.yml](https://github.com/sider/runners/blob/master/goodcheck.yml) as a separate PR?

Reply:
Thank you. I changed it to `one_or_more_strings?`. 

Sure. Adding the goodcheck rule is a good idea.

Reply:
Rule added and merged here https://github.com/sider/runners/pull/2675